src/prefect/environments/execution/local.py

```diff
@@ -17,15 +17,19 @@ class LocalEnvironment(Environment):
             Agents when polling for work
         - on_start (Callable, optional): a function callback which will be called before the flow begins to run
         - on_exit (Callable, optional): a function callback which will be called after the flow finishes its run
+        - metadata (dict, optional): extra metadata to be set and serialized on this environment
     """
 
     def __init__(
```

Comment:
Since this `__init__` only forwards to its base class, the whole `__init__` could be removed.